lib/dolly/request.rb

```diff
@@ -101,6 +102,16 @@ def full_path resource
     def attachment_path resource, attachment_name
       "#{full_path(resource)}/#{attachment_name}"
     end
+
+    def log_request resource, response_code
+      log_value = ->(resource, response_code) { "Query: #{resource}, Response Code: #{response_code}" }
+      case response_code
+      when 200..399
+        Dolly.logger.info log_value[resource, response_code]
+      when 400..600
+        Dolly.logger.warn log_value[resource, response_code]
```

Comment:
Im not sure either.  In the end, I feel as if the user will be adequately warned/given an error https://github.com/amco/dolly/blob/1.0/lib/dolly/request.rb#L69.  

I can Change it to error if you feel strongly.  Regardless, this is a key candidate for a config option as well, which can be expanded upon in a later PR.